freud/order.pyx

```diff
@@ -568,6 +570,8 @@ cdef class Steinhardt(_PairCompute):
 
     def __cinit__(self, l, average=False, wl=False, weighted=False,
                   wl_normalize=False):
+        if not hasattr(l, "__len__"):
```

Comment:
Use `isinstance(l, collections.abc.Sequence)`.

Reply:
For some reason at the time I was trying to avoid the import, but will do.